File: chpt 3 assign translated to java/NestedControl.java
import java.util.Scanner;

public class NestedControl {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double largest = Double.NEGATIVE_INFINITY;
        double secondLargest = Double.NEGATIVE_INFINITY;

        for (int row = 0; row < 10; row++) {
            System.out.print("Enter number " + (row+1) + ": ");
            double num = scanner.nextDouble();

        if (num > largest) {
                secondLargest = largest;
                largest = num;
            } else if (num > secondLargest) {
                secondLargest = num;
            }
        }
	System.out.println("The two largest values are " + largest + " and " + secondLargest);
    }
}

File: python chpt 3 assignments/nestedcontrol.py
largest = float('-inf')
second_largest = float('-inf')

for i in range(10):
    num = float(input(f"Enter number {i+1}: "))

    if num > largest:
        second_largest = largest
        largest = num
    
    elif num > second_largest:
        second_largest = num

print(f"The two largest values are {largest} and {second_largest}")

